### ISSUE TITLE
Add/remove student list is always on the DOM, even when hidden

### ISSUE THREAD

**Issue by bwreid** [WikiEducationFoundation/WikiEduDashboard#2502]
The list of students that visually appears when an instructor clicks "Add/Remove students" on the Students tab is always on the page even if it's not visible. This causes issues for screen readers in that it still senses those students names and will try and read them off.

It looks as though the students dropdown is "hidden" due to opacity. Instead, we could modify the CSS to use a better property (e.g. visible) or actually remove the pop-up from the page and have it reappear on click.
